apps/functions/deadline-submissions/deadline-submission-command/back-office-api-client.js

```diff
@@ -10,7 +10,7 @@ import config from './config.js';
 async function getCaseID(caseReference) {
 	try {
 		const result = await got
-			.get(`https://${config.apiHost}/applications?reference=${caseReference}`)
+			.get(`https://${config.apiHost}/applications/reference/${caseReference}`)
 			.json();
 
 		return result.id;
```

Comment:
@HadrianHughesKC, I changed this. 
Not sure how to test this function specifically, but I tested the API and still works